app/views/application_letters/check.html.erb

```diff
@@ -25,7 +23,11 @@
   <tr>
     <th><%= ApplicationLetter.human_attribute_name(:motivation) %>:</th>
     <td id="application-motivation" data-orig-text="<%= @application_letter.motivation %>">
-      <%= truncate @application_letter.motivation, length: 300 do link_to I18n.t('events.list.read_on'), 'javascript:jQuery("#application-motivation").text(jQuery("#application-motivation").attr("data-orig-text"))' end %>
+      <%= truncate @application_letter.motivation, length: 300 do
+        link_to I18n.t('events.list.read_on'),
+          'javascript:jQuery("#application-motivation")
+            .text(jQuery("#application-motivation").attr("data-orig-text"))'
```

Comment:
I know I'm picky here, but wouldn't it make sense to either let the javascript thing stay in one line or insert a line break after every dot?